src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -329,16 +329,20 @@ export function createApiFactory(
 				return extHostTerminalService.onDidCloseTerminal(listener, thisArg, disposables);
 			},
 			showInformationMessage(message, first, ...rest) {
-				return extHostMessageService.showMessage(Severity.Info, message, first, rest);
```

Comment:
These changes are the main ones I'm uncertain about. Let me know if you think a different fix would be better